app/application.py

```diff
@@ -67,19 +67,33 @@ def convert(link, model):
         return
 
 
-def write_to_file(result, url, title, date):
+def write_to_file(result, url, title, date, tags, category, speakers):
     transcribed_text = result[0]
     video_title = result[1][:-4]
     if title:
         file_title = title
     else:
         file_title = video_title
-
+    tags = tags.strip()
+    tags = tags.split(",")
+    for i in range(len(tags)):
+        tags[i] = tags[i].strip()
+    speakers = speakers.strip()
+    speakers = speakers.split(",")
+    for i in range(len(speakers)):
+        speakers[i] = speakers[i].strip()
+    category = category.strip()
```

Comment:
This line crashed on me when I didn't pass in a category

<img width="855" alt="Screen Shot 2023-01-18 at 8 18 44 PM" src="https://user-images.githubusercontent.com/755825/213333297-bb2821df-0a05-4862-8d12-40d86611e569.png">

Reply:
Fixed the bug category was expecting a string but was getting None.